### PR TITLE
Improvements to the API.md

### PR DESCRIPTION
**Pull request by devinstewart**
To address #15 

@lloydbenson / @devinivy / @geek / @cjihrig -- Here is my Take 2 at the PR.

Thank you for your patience and understanding.

### CODE REVIEW

**Review by lloydbenson — APPROVED** [review]
Looks better! I'll wait for others to review it today but I'll merge tomorrow and push this out on our website if I see no objections. Thank you for this much needed contribution!

**Review by devinivy — APPROVED** [review]
This is a fantastic base for the documentation of this module— much appreciate the contribution!

File: API.md [hunk 1]
```diff
@@ -1,99 +1,408 @@
...
+                algorithms: ['HS512'],
+                kid: 'someKid2'
+            }
+        ]
+    }
+    ...
+
+    // Single Public Key
+    {
+        keys: `fs.readFileSync('public.pem')`
```
Comment (devinivy) [inline API.md:134]:
```suggestion
        keys: fs.readFileSync('public.pem')
```

File: API.md [hunk 2]
```diff
@@ -1,99 +1,408 @@
...
+                isValid: true,
+                credentials: { user: artifacts.decoded.payload.user }
+            };
+        }
+    });
+
+    // Set the strategy
+
+    server.auth.default('my_jwt_stategy');
 };
```
Comment (devinivy) [inline API.md:51]:
```suggestion
};

internals.start();
```

**Pull request merged**